tests/Extensions/Logging.UnitTests/ServiceCollectionTests.cs

```diff
@@ -48,6 +49,17 @@ public void AddOmexLoggerOnLogBuilder_RegistersLogger()
 			ValidateTypeRegistration<ILogger<ServiceCollectionTests>>(builder.Services);
 		}
 
+		[TestMethod]
+		public void AddOmexLoggerOnLoadInitialLogger_RegistersLogger()
+		{
+			ILoggerFactory factory = LoggerFactory.Create(builder => {
+				builder.LoadInitialLogger();
+			});
+
+			ILogger logger = factory.CreateLogger<OmexLogger>();
+			Assert.IsNotNull(logger);
+		}
+
 		private T ValidateTypeRegistration<T>(IServiceCollection collection)
 			where T : class
```

Comment:
Here a sample of checking that events reported to event source https://github.com/microsoft/Omex/blob/master/tests/Extensions/Logging.UnitTests/OmexLogEventSenderTests.cs
But we would need to check that running in pararel with other test won't interfere (since source is static) so maybe you could just extend existing test to check that initial logger also createas events